appender.go

```diff
@@ -40,23 +101,32 @@ func NewAppenderFromConn(driverConn driver.Conn, schema, table string) (*Appende
 		panic("database/sql/driver: misuse of duckdb driver: Appender after Close")
 	}
 
-	var schemastr *(C.char)
+	var cSchema *C.char
 	if schema != "" {
-		schemastr = C.CString(schema)
-		defer C.free(unsafe.Pointer(schemastr))
+		cSchema = C.CString(schema)
+		defer C.free(unsafe.Pointer(cSchema))
 	}
 
-	tablestr := C.CString(table)
-	defer C.free(unsafe.Pointer(tablestr))
+	cTable := C.CString(table)
+	defer C.free(unsafe.Pointer(cTable))
 
-	var a C.duckdb_appender
-	if state := C.duckdb_appender_create(*dbConn.con, schemastr, tablestr, &a); state == C.DuckDBError {
-		return nil, fmt.Errorf("can't create appender")
-	}
+	var appender C.duckdb_appender
+	state := C.duckdb_appender_create(*dbConn.con, cSchema, cTable, &appender)
```

Comment:
Maybe we should shorten `dbConn` simply to `conn` as the corresponding struct field in the `Appender` is also called `c`.